diff --git a/dist/index.cjs b/dist/index.cjs
index 9d45d971c34cf3805b2b37c6cd0db7b5842eee4b..2203722dd06154da83df8f90688587f68fa0212a 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -35,8 +35,8 @@ const VIRTUAL_ENTRY_ALIAS = [
   /^(?:virtual:)?uno(?::(.+))?\.css(\?.*)?$/
 ];
 const LAYER_MARK_ALL = "__ALL__";
-const RESOLVED_ID_WITH_QUERY_RE = /[\/\\]__uno(?:(_.*?))?\.css(\?.*)?$/;
-const RESOLVED_ID_RE = /[\/\\]__uno(?:(_.*?))?\.css$/;
+const RESOLVED_ID_WITH_QUERY_RE = /__uno(?:(_.*?))?\.css(\?.*)?$/;
+const RESOLVED_ID_RE = /__uno(?:(_.*?))?\.css$/;
 function resolveId(id) {
   if (id.match(RESOLVED_ID_WITH_QUERY_RE))
     return id;
diff --git a/dist/index.mjs b/dist/index.mjs
index 9780cf2901dab372b703448bb8bd6813561204ec..0698f8f537988055503da2ce0a0aff9cc601f9b4 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -20,8 +20,8 @@ const VIRTUAL_ENTRY_ALIAS = [
   /^(?:virtual:)?uno(?::(.+))?\.css(\?.*)?$/
 ];
 const LAYER_MARK_ALL = "__ALL__";
-const RESOLVED_ID_WITH_QUERY_RE = /[\/\\]__uno(?:(_.*?))?\.css(\?.*)?$/;
-const RESOLVED_ID_RE = /[\/\\]__uno(?:(_.*?))?\.css$/;
+const RESOLVED_ID_WITH_QUERY_RE = /__uno(?:(_.*?))?\.css(\?.*)?$/;
+const RESOLVED_ID_RE = /__uno(?:(_.*?))?\.css$/;
 function resolveId(id) {
   if (id.match(RESOLVED_ID_WITH_QUERY_RE))
     return id;